app/models/concerns/attribute_displayable.rb

```diff
@@ -11,4 +11,14 @@ def other_phone_display
     return nil unless other_phone.present?
     "#{other_phone[0..2]}-#{other_phone[3..5]}-#{other_phone[6..9]}"
   end
+
+  def appointment_date_display
+    return nil unless appointment_date.present?
+    "#{appointment_date.strftime("%m-%d-%Y")}"
+  end
+
+  def procedure_date_display
+    return nil unless fulfillment.procedure_date.present?
+    "#{fulfillment.procedure_date.strftime("%m-%d-%Y")}"
```

Comment:
You might be able to use `display_date` on these -- I think mongoid saves everything as timestamps.